docs/src/pages.js

```diff
@@ -179,8 +179,8 @@ const pages = [
       { pathname: '/guides/composition' },
       { pathname: '/guides/server-rendering' },
       { pathname: '/guides/responsive-ui', title: 'Responsive UI' },
-      { pathname: '/guides/migration-v3', title: 'Migration From v3 to v4' },
-      { pathname: '/guides/migration-v0x', title: 'Migration From v0.x to v1' },
+      { pathname: '/guides/migration-v3', title: 'Migration From v3' },
```

Comment:
The context is important here, I think that we should keep it

Reply:
I don’t disagree, but someone already changed it in the translations - this just brings `pages.js` in line so that it doesn’t get put back when `docs:i18n` Is run.

Reply:
Oh, that might have been me 😂. The changes look good to me.